codecept.conf.js

```diff
@@ -0,0 +1,52 @@
+var gruntConfig = require('./grunt.conf')()
+
+var options = gruntConfig.connect.options
+var proxy = gruntConfig.connect.proxies[0]
+
+module.exports.config = {
+  tests: './test/e2eTests/**/*_test.js',
+  timeout: 10000,
+  output: './test/e2eTests/output',
+  helpers: {
+    Puppeteer: {
+      url: `http://${options.hostname}:${options.port}`,
+      driver: 'hosted',
+      browser: 'chrome',
+      rootElement: 'body',
+      show: true,
+    },
+    REST: {
+      endpoint: `http://${proxy.host}${proxy.context}/ws/rest/v1`,
```

Comment:
Had to add `proxy.port` here.

Reply:
Oops. It worked without the port. Thanks for catching. Added the port in.